src/app.js

```diff
@@ -2,10 +2,14 @@ import winston from 'winston';
 import express from 'express';
 import logging from './helpers/logging';
 import routes from './routes/index';
-
+import './config/cloudinary.config';
+import { mock } from './middlewares/validators/socialLogin-mock';
 
 const app = express();
 
+if (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test') {
```

Comment:
this is also already merged

Reply:
Tests were failing without it and it wasn't there. So we decided to add it.